help/styles/styles.css

```diff
@@ -514,10 +514,11 @@ main .section.rating .default-content-wrapper {
   width: unset;
 }
 
-main .section.rating .default-content-wrapper h3 {
+main .section.rating .default-content-wrapper h2 {
```

Comment:
should we keep both `h2` and `h3` in case some H3 slip through?

Reply:
no, I have fixed in import that it will always be h2. We will keep this in the authoring guidelines too so their authors are also aware.. else accessibility issue comes in